src/minErr.js

```diff
@@ -36,17 +36,27 @@ function minErr(module, ErrorConstructor) {
     var SKIP_INDEXES = 2;
 
     var templateArgs = arguments,
+      argsDebuggedStringIndexes = [],
       code = templateArgs[0],
       message = '[' + (module ? module + ':' : '') + code + '] ',
       template = templateArgs[1],
       paramPrefix, i;
 
+    function getTemplateArgAndConvertToDebugStringFormat(index) {
```

Comment:
I tried to Improve it because i found that toDebug method is called 2 times for every argument times 
One inside the template replace callback

    message += template.replace(/\{\d+\}/g, function(match) {
        var index = +match.slice(1, -1),
          shiftedIndex = index + SKIP_INDEXES;
  
        if (shiftedIndex < templateArgs.length) {
         return toDebugString(templateArgs[shiftedIndex]);
        }

and during encoding the url

    for (i = SKIP_INDEXES, paramPrefix = '?'; i < templateArgs.length; i++, paramPrefix = '&') {
        message += paramPrefix + 'p' + (i - SKIP_INDEXES) + '=' +
         encodeURIComponent(toDebugString(templateArgs[i]));
      } 

So toDebugString is doing a lot of work (copying and stringifying), i thought if it would be better if i cache the debugged string since i will use it again during encoding the url.

you are right 
"this is an exceptional situation (we are throwing an error), so performance is not a concern."

Do you want me to undo these changes ?